app/lib/active_merchant_hacks.rb

```diff
@@ -2,6 +2,9 @@
 
 # ActiveMerchant monkey patches
 
+module ActiveMerchantHacks
+end
```

Comment:
I would rather move this file out of autoload path. Or better do same as with `HashHacks`.